ilan/build

```diff
@@ -23,5 +23,14 @@ fi
 
 docker build -t installation-manager im || exit $?
 docker run --rm -v $(pwd):/host installation-manager /host/install_ihs $1 $2 $3 || exit $?
-#docker build -t websphere-traditional:install install || exit $?
-#docker build -t websphere-traditional:profile profile || exit $?
+
+docker run -d --name tar_server -v $(pwd)/ihs$1.tar.gz:/host/ihs$1.tar.gz -w /host python:2-slim python -m SimpleHTTPServer
+tar_server_ip=$(docker inspect --format '{{ .NetworkSettings.IPAddress }}' tar_server)
+tar_url="http://${tar_server_ip}:8000/ihs${1}.tar.gz"
+
+
+# Build install image from hosted tar file
```

Comment:
I don't think "install image" makes sense in the context of the IHS image. Just say "image". The comment is also pretty superfluous given the echo on the line after.